transcription/transcription.go

```diff
@@ -76,3 +77,17 @@ func fileNameFromURL(url string) string {
 	fileName := tokens[len(tokens)-1]
 	return fileName
 }
+
+// Task returns a task function for transcription using transcription functions.
+func Task(audioURL string, emailAddresses []string) func() error {
```

Comment:
Test this function == mock tests?

not testing `fileNameFromURL`

waiting to test `DownloadFileFromURL` because issues with testing `http.Get`